docs/javascript-action.md

```diff
@@ -6,57 +6,55 @@ The [javascript-template](https://github.com/actions/javascript-template) repo c
 
 Navigate to https://github.com/actions/javascript-template
 
-Click on `Use this template` to create the repo for your action.
+Click on `Use this template` to create the repo for your action.  Provide a name such as `myaction`.
 
 ![template](assets/node12-template.png)
 
-Complete creating your repo and clone the repo.
+# Clone and Update
 
-> NOTE: The location of the repo will be how users will reference your action in their workflow file with the using keyword.
-
-e.g. To use https://github.com/actions/setup-node, user's will author:
-
-```yaml
-steps:
-    uses: actions/setup-node@master
+```bash
+$ git clone <repolocation>
+$ cd myaction
 ```
 
+Update the `author` element in the package.json file.
+
 # Dev Workflow
 
 The workflow below describes one possible workflow with a branching strategy.  Others exist.  
 
-> Key Point: the branch that users reference in their workflow files should reference an action that has **only** the production dependencies. 
+> Key Point: the branch that users reference in their workflow files should reference an action from a distribution branch that has **only** the production dependencies. 
 
-The workflow below describes a strategy where you code in master (with node_modules ignored) with a v1 branch users reference and contains the product references.  Actions are self contained referenced on the github graph of repos.
+The workflow below describes a strategy where you code in master (with node_modules ignored) with a distribution releases/v1 branch users reference via a tag.  Actions are self contained referenced from the github graph of repos, downloaded by the runner and run intact at runtime.
 
 ## Install Dependencies
 
-After creating a repo from the template and cloning it, you will be in master.  The command below will install the toolkit, other dependencies and dev dependencies
+After creating a repo from the template and cloning it, you will be in master.  The command below will install the toolkit, other dependencies and dev dependencies.  node_modules are ignored in the coding master branch.
 
 ```bash
 $ npm install
 ```
 
 ## Define Metadata
 
-Your action has a name and a description.  Update the author.
-
-Create inputs that your unit of work will need.  These will be what workflow authors set with the `with:` keyword.
+Your action has a name and a description.  Update all fields .
 
 ```yaml
-name: 'My new action'
-description: 'A test action'
-author: 'GitHub'
+name: 'Hello'
+description: 'Outputs Hello to a named input'
+author: 'me'
 inputs: 
-  myInput:
-    description: 'Input to use'
-    default: 'world'
+  name:
+    description: 'the name to say hello to'
+    default: 'World'
 runs:
   using: 'node12'
   main: 'lib/main.js'
 
 ```
 
+The `name` input will be referenced by workflow authors using the `with:` keyword.
```

Comment:
Add example?

Reply:
Yeah, it's covered in the `Users referencing` section further down in this doc but it's not using the `with` keyword.  I will update this blurb to mention that and update that section to make it clearer.

Reply:
Good catch ...